cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -239,6 +241,12 @@ func main() {
 		"Whether to use the legacy gke_container MonitoredResource type instead of k8s_container").
 		Default("false").BoolVar(&cfg.useGkeResource)
 
+	a.Flag("stackdriver.store-in-files", "Whethere to store payload in filesystem. For use in troubleshooting.").
+		Default("false").BoolVar(&cfg.storeInFiles)
+
+	a.Flag("stackdriver.store-in-files-directory", "Directory to store payload. For use in troubleshooting.").
+		Default(os.TempDir() + "/CreateTimeSeriesRequest").StringVar(&cfg.storeInFilesDirectory)
```

Comment:
Alternatively think of not setting a default flag and remove the boolean above. Fewer flags is better, and I'm not sure using tempdir is a useful default.